SDCTA/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { NavbarLogin } from './NavbarLogin';
+import { NavbarSubscribe } from './NavbarSubscribe';
+import { NavbarNoButton } from './NavbarNoButton';
+import './Navbar.css';
+import logo from "./SDTEF Logo Transparent Background 1.svg";
+
+export const Navbar = () => {
+   const [isFree, setIsFree] = useState(false);
```

Comment:
We probably want this state to be in redux state and then use the useSelector hook. See this [page](https://redux.js.org/tutorials/essentials/part-2-app-structure#:~:text=A%20%22slice%22%20is%20a%20collection,multiple%20%22slices%22%20of%20state.) and the counter files (particularly counterSlice.ts) for an example. 

We'll want one slice called UserStatus or something that has the reducers to update these states. Using createSlice like in the example basically auto-create the actions and selectors for you.

Reply:
We can probably meet on zoom or in-person to do this together if you'd like since it is confusing the first time you do it.